stock_report_quantity_by_location/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_stock_report_quantity_by_location_prepare,access_stock_report_quantity_by_location_prepare,model_stock_report_quantity_by_location_prepare,base.group_user,1,1,1,1
+access_stock_report_quantity_by_location,access_stock_report_quantity_by_location,model_stock_report_quantity_by_location,base.group_user,1,1,1,1
```

Comment:
Change the group to stock.group_stock_user (Inventory User) and the permission to 1,1,1,0 (standard permission in any wizard in the core).

Reply:
Done.